### ISSUE TITLE
aoscx_checkpoint: GENERIC OPERATION ERROR: 404 Not Found: Code: 404

### ISSUE THREAD

**Issue by rhizoet** [aruba/aoscx-ansible-collection#33]
When I want to create a checkpoint, I get the following error:
`pyaoscx.exceptions.generic_op_error.GenericOperationError: 'GENERIC OPERATION ERROR: 404 Not Found: Code: 404'`

A few months ago this worked without problems.

In the playbook I try to run the following:

```
- name: Copy running-config to startup-config
  arubanetworks.aoscx.aoscx_checkpoint:
    source_config: "running-config"
    destination_config: "startup-config"
```

I'm using the latest collection and pyaoscx.

**Comment by tchiapuziowong** [comment]
Hi @rhizoet can you share which switch model/platform and firmware you're using?

**Comment by rhizoet** [comment]
Hi @tchiapuziowong ,

sorry for the late reply. The model/platform: HPE Aruba JL479A 8320 and the firmware: TL.10.08.1050

**Comment by tchiapuziowong** [comment]
This appears to be a bug and we're working internally to resolve - will update issue when fix has been pushed

**Comment by tchiapuziowong** [comment]
Hey @rhizoet great news!
We've officially published the release of v4.1.1 of the AOS-CX Ansible collection, the fix for this issue as well as others were a part of the release so please install the latest version of the collection and our Python SDK using the commands below:
`ansible-galaxy collection install arubanetworks.aoscx -f`
` pip3 install pyaoscx --upgrade`

Please let us know if you discover any further issues or unwanted behavior and thank you again for your patience!

**Comment by rhizoet** [comment]
Hey @tchiapuziowong thanks for your reply.
I've tested the new release and it works as expected. The checkpoint is created again without error messages.

Only the check mode is missing and I am happy ;-)

Thanks a lot for the great work.
